utils/utils.py

```diff
@@ -134,27 +134,23 @@ def identify_category(self, page, i, path_to_new_directory):
         """Identify the category of the page."""
         rect = fitz.Rect(60, 30, 200, 60)
         if i % 2 == 0:
-            rect = fitz.Rect(400, 30, 580, 60)
-        rect_dict = {"left": rect, "top": rect, "right": rect, "bottom": rect}
+            rect = fitz.Rect(450, 55, 580, 350)
+        left, top, right, bottom = rect
         if self.debug:
-            pix = page.get_pixmap(
-                clip=(
-                    rect_dict["left"],
-                    rect_dict["top"],
-                    rect_dict["right"],
-                    rect_dict["bottom"],
-                )
-            )
+            pix = page.get_pixmap(clip=(left, top, right, bottom))
+            pix.save(f"page-{page.number}-category_side.png")
+        if self.debug:
+            pix = page.get_pixmap(clip=(left, top, right, bottom))
```

Comment:
the pix should be only saved in debug mode.

Reply:
But they are only save in debug mode, aren't they? Only irritating way I see is two same if-cases